views/partials/currently-playing.ejs

```diff
@@ -43,4 +45,26 @@
       {{user}}
     </span>
   </div>
+
+  <!-- CHAT -->
+  <div class="extra content">
+    <span style="cursor: pointer;" ng-click="toggleChat()">
+      {{chats.length}} message{{chats.length != 1 ? 's' : ''}}
+      <i class="blue chat caret right icon"></i>
+      <i class="blue chat caret down icon" style="display: none;"></i>
+    </span>
+  </div>
+  <div class="chat extra content" style="display: none;">
+    <div id="chat-list" class="ui relaxed list" style="overflow-y: auto;">
+      <div class="item" ng-repeat="chat in chats | orderBy: time track by $index">
+        <div class="content">
+          <div class="header">{{chat.message}}</div>
+          <div class="description">{{chat.username}}</div>
```

Comment:
Having the username under the message seemed a bit odd to me, I expected it to be above.